src/js/components/DialogsComponent.jsx

```diff
@@ -40,7 +41,9 @@ var DialogsComponent = React.createClass({
       DialogEvents.CONFIRM_SHOW,
       this.onDialogConfirmShow
     );
-    DialogStore.removeListener(DialogEvents.CONFIRM_DISMISS, this.onDialogClose);
+    DialogStore.removeListener(
+      DialogEvents.CONFIRM_DISMISS, this.onDialogClose
```

Comment:
This looks like a mistake on my part, I would normally have done the latter of those two options. 

In any case, I don't really like former. I would prefer simply to insert a single line break eg:
```
DialogStore.removeListener(DialogEvents.CONFIRM_DISMISS, 
  this.onDialogClose);
```

Otherwise the close-parenthesis gets its own line but the open-parenthesis doesn't, and it looks wonky.